crates/holochain_types/src/app.rs

```diff
@@ -483,6 +486,7 @@ impl InstalledAppCommon {
             agent_key,
             role_assignments,
             manifest,
+            installed_at: Timestamp::now(),
```

Comment:
I'd prefer if the timestamp were passed in as an arg:

1. to make sure we don't construct type this outside of actual app installation and use that moment as the timestamp.
2. to give a better test, since it's possible for your test to have apps with the same timestamp.

Reply:
If you just throw a `#[cfg(feature = "test_utils")]` on new_legacy I think we can call that good without adding a timestamp arg.

Reply:
okay added the feature gate on new_legacy